### DISCUSSION TITLE
Amend border colour with JS - not working

### DISCUSSION THREAD

**Discussion by kempster** [tailwindlabs/tailwindcss#9120] [Help]
The following is not working. The button is toggling the "border-black" class, but this is not rendering on the page. If I set "border-black" on the HTML, it works fine. Am I missing a trick?

```
module.exports = {
  theme: {
    colors: {
      'black': '#000000',
      'yellow': '#FEDB00'
    },
    borderColor: {
      DEFAULT: '#CDCDCD',
      'black': '#000000'
    }
  }
}
```

```
<button data-tag class="border hover:border-black">...</button>
```

```
const tag = document.querySelector('[data-tag]');

tag.onclick = () => {
    tag.classList.toggle('bg-yellow');
    tag.classList.toggle('border-black');
};
```

**Comment by wongjn** [comment]
Is the JavaScript code scanned by Tailwind? In other words, is it covered by the `content` glob(s) in `tailwind.config.js`? If the `border-black` class is being applied but styling is not working, then it suggests that Tailwind is not generating the class.
